server/middleware/auditMiddleware.ts

```diff
@@ -80,3 +124,12 @@ export default function auditMiddleware({ auditService }: Services): Router {
 
   return router
 }
+
+// Checks page view has been auditted, if no audit event has been raised router will be skipped
+export function checkPageViewAuditted(router: Router) {
+  router.get('*', (req: Request, res: Response, next: NextFunction) => {
+    if (res.locals.auditPageViewEvent || res.locals.auditPageViewEvent === null) return next()
+    logger.error(`No audit event found for route, "${req.path}". Skipping router.`)
+    return next('router')
+  })
+}
```

Comment:
This isn't an ideal solution, but because express has no way to know whether a route will later match another route our only option to catch unaudited routes without transforming all 404s into 500 errors is to treat them as unconfigured routes and display a 404. This will allow us to catch pages which are unaudited and ensure every page is audited.